gradoop-examples/src/main/java/org/gradoop/utils/sampling/RandomVertexNeighborhoodSamplingRunner.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.utils.sampling;
+
+import org.apache.flink.api.common.ProgramDescription;
+import org.gradoop.examples.AbstractRunner;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.operators.sampling.RandomVertexNeighborhoodSampling;
+import org.gradoop.flink.model.impl.operators.sampling.functions.Neighborhood;
+
+/**
+ * Runs {@link RandomVertexNeighborhoodSampling} for a given
+ * graph and writes the sampled output.
+ */
+public class RandomVertexNeighborhoodSamplingRunner extends AbstractRunner implements ProgramDescription {
+
+  /**
+   * Runs the {@link RandomVertexNeighborhoodSampling} operator
+   * on the specified graph and writes the result to the specified output.
+   * <p>
+   * args[0] - path to input graph
+   * args[1] - format of input graph (csv, json, indexed)
+   * args[2] - path to output graph
+   * args[3] - format of output graph (csv, json, indexed)
+   * args[4] - sampling threshold
+   * args[5] - type of neighborhood (input, output, both)
+   *
+   * @param args arguments
+   */
+  public static void main(String[] args) throws Exception {
+    Neighborhood.NeighborType neighborType = null;
```

Comment:
For future examples we could think about using the [`ParameterTool`](https://ci.apache.org/projects/flink/flink-docs-release-1.5/dev/best_practices.html#parsing-command-line-arguments-and-passing-them-around-in-your-flink-application) of Flink to easily validate input arguments.